frontend/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# TODO: can't use vanilla alpine version since python is needed for gql-codegen stuff.  Could consider installing just python if the image size of node is too big
+FROM node AS builder
+
+WORKDIR /app
+
+COPY . /app/
+COPY src /app/src
+COPY static /app/static
+
+RUN npm i && npm run build
+
+FROM node
```

Comment:
can't we use alpine here? now that the gql code is built we don't need python anymore.

Reply:
that's a good thought since the dep won't be utilized anymore.  I'll double-check locally and make that change if still works.